docs/config.json

```diff
@@ -419,13 +419,33 @@
       "title": "Manage your Cluster",
       "entries": [
         {
-          "title": "Admin Guides",
-          "slug": "/management/admin/",
+          "title": "Registering Teleport Services",
+          "slug": "/management/register-services/",
           "entries": [
             {
-              "title": "Adding Nodes",
-              "slug": "/management/admin/adding-nodes/"
+              "title": "Via a Join Token",
+              "slug": "/management/register-services/register-services-join-token/"
+            },
+            {
+              "title": "Via AWS EC2",
+              "slug": "/management/register-services/register-services-aws-ec2/",
+              "forScopes": ["oss", "enterprise"]
+            },
+            {
+              "title": "Via AWS IAM",
+              "slug": "/management/register-services/register-services-aws-iam/"
             },
+            {
+              "title": "Via a Token Using the Auth Service",
```

Comment:
Friendly ping for @atburke who is working on Azure joining and should be aware of this reorganization.